system/ntp/detectors-ntp.tf

```diff
@@ -0,0 +1,17 @@
+resource "signalfx_detector" "ntp" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] System ntp"
+
+  program_text = <<-EOF
+        signal = data('ntp.offset_seconds', filter=${module.filter-tags.filter_custom})${var.ntp_aggregation_function}.${var.ntp_transformation_function}(over='${var.ntp_transformation_window}').publish('signal')
+        detect(when(signal > ${var.ntp_threshold_warning})).publish('WARN')
+  EOF
+
+  rule {
+    description           = "is too high > ${var.ntp_threshold_warning}"
+    severity              = "Warning"
+    detect_label          = "WARN"
+    disabled              = coalesce(var.ntp_disabled_warning, var.ntp_disabled, var.detectors_disabled)
```

Comment:
while there is only one severity I think you can delete `ntp_disabled_warning` variable and its usage

Reply:
ok done